tools/cas/BUILD

```diff
@@ -6,6 +6,7 @@ go_library(
     importpath = "github.com/buildbuddy-io/buildbuddy/tools/cas",
     visibility = ["//visibility:private"],
     deps = [
+        "//cli/storage",
```

Comment:
These changes seem to have leaked in from the other PR

Reply:
Sorry it was a bad git rebase locally that updated the wrong branch.

PTAL.